### PR TITLE
WIP - Update the content for Nonprofit pages/review-entity-type/nonprofit.js (Do Not Merge)

### PR DESCRIPTION
**Pull request by tinapc**
## What Does This Do?
- Update the content for Nonprofit pages/review-entity-type/nonprofit.js

### CODE REVIEW

**Comment by tinapc** [comment]
@krzysiekstefanski  Could you plz take a review? I can show you a video to preview if you dont have a chance to see what I did done.

File: src/atomic/organisms/accordion/accordion-with-counting.js [hunk 1]
```diff
@@ -92,14 +92,46 @@ const PanelWrapper = styled.div`
...
 
-    li {
-      padding-top: 16px;
+  //   li {
+  //     padding-top: 16px;
+  //   }
+  // }
+`;
+
+const ListItems = styled.div`
```
Comment (Riotters) [inline src/atomic/organisms/accordion/accordion-with-counting.js:105]: ListItems should be ul, there's no need to create another div.

File: src/atomic/atoms/styles/colors.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ export const color = {
   grey3: "#DDDDDD",
   grey4: "#F4F4F4",
   grey5: "#FCFCFC",
+  grey6: "#63666e",
```
Comment (Riotters) [inline src/atomic/atoms/styles/colors.js:8]: Palette is already done, use closest color.

File: src/atomic/sections/general/cta.js [hunk 3]
```diff
@@ -103,13 +103,13 @@ const Cta = () => (
...
         <VisibilitySensor partialVisibility once>
-          {({ isVisible }) => <h2 className={isVisible ? "slideUp enter" : "slideUp"}>Choose the right and&nbsp;start your business</h2>}
+          {({ isVisible }) => <h2 className={isVisible ? "slideUp enter" : "slideUp"}>{cta.header || "Choose the right and&nbsp;start your business"}</h2>}
         </VisibilitySensor>
         <VisibilitySensor partialVisibility once>
           {({ isVisible }) => <p className={isVisible ? "slideUp enter" : "slideUp"}>$0 + State Fee & FREE Registered Agent for 1st year.</p>}
         </VisibilitySensor>
         <VisibilitySensor partialVisibility once>
-          {({ isVisible }) => <Button className={isVisible ? "slideUp enter" : "slideUp"} content={{ text: `Start Now`, url: `/` }} theme="primary56" arrow marginSM="0 auto" />}
+          {({ isVisible }) => <Button className={isVisible ? "slideUp enter" : "slideUp"} content={{ text: `${cta.button.text || `Start Now`}`, url: `${cta.button.url || `#`}` }} theme="primary56" arrow marginSM="0 auto" />}
```
Comment (Riotters) [inline src/atomic/sections/general/cta.js:112]: For now, all urls should lead to the home page.

File: src/static/review-entity-type/c-corporation.js [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
-  header: `Registered Agent Service`,
-  text: `Dependable, Easy, and Free for the First Year When You Incorporate with Incfile. A nationwide Registered Agent service provider you can trust at an affordable price. Available in all U.S. states.`,
+  header: `Is a C Corporation Right For Your Business?`,
+  text: `What you need to know about forming a C Corporation`,
   button: [
     {
-      text: "Sign up now",
-      url: "/",
+      text: "Form Your C Corp Now",
+      url: "https://www.incfile.com/form-order-now.php?entityType=CCorporation",
```
Comment (Riotters) [inline src/static/review-entity-type/c-corporation.js:7]: Sholud stay "/" for now.

File: src/static/review-entity-type/nonprofit.js [hunk 5]
```diff
@@ -1,140 +1,214 @@
...
-    text: `launch your business`,
-    url: `/`,
-  },
+	header: `Launch Your Business`,
+	header2: `your business with Incfile`,
+	text: `No Contracts. No Surprise.`,
+	text2: `Only $0 + State Fee to Launch Your Business.`,
+	button: {
+		text: `Start Now`,
+		url: `https://www.incfile.com/form-order-now.php?entityType=Nonprofit`,
```
Comment (Riotters) [inline src/static/review-entity-type/nonprofit.js:180]: "/" :)

File: src/static/review-entity-type/nonprofit.js [hunk 6]
```diff
@@ -1,140 +1,214 @@
...
+	},
 };
 
+export const cta = {
+  header: `Launch Your Business Only $0 + State Fee.`,
+  button: {
+    text: `See Detailed Pricing`,
+    url: `https://www.incfile.com/form-order-now.php?entityType=Nonprofit`,
```
Comment (Riotters) [inline src/static/review-entity-type/nonprofit.js:188]: "/" :)

File: src/components/about-tabs.js [hunk 7]
```diff
@@ -260,36 +263,61 @@ const AboutTabs = ({layout, columns}) => (
...
+                <h3>What Is a Nonprofit Corporation?</h3>
+                <p>A nonprofit corporation is an organization whose purpose is something other than making a profit. A nonprofit donates its revenue to achieve a specific goal that benefits the public, instead of distributing it to shareholders. There are over 1.5 million nonprofit organizations registered in the US.</p>
+                
+                <p>Being a nonprofit does not mean the organization won’t make a profit. Nonprofits can make money, but all of the money must go back into the organization by paying employee salaries, administrative expenses and other overhead costs. No one person or group owns a nonprofit. Assets from a nonprofit can be sold, but it benefits the whole organization rather than individuals.</p>
+                <p>While anyone can incorporate as a nonprofit, only those who pass the stringent standards set forth by the government can achieve tax exempt, or 501c3, status. The process of creating incorporation papers is just the first step in becoming a completely recognized nonprofit organization. We discuss the steps to becoming a nonprofit further into this page.</p>
```
Comment (Riotters) [inline src/components/about-tabs.js:270]: import text from external, change to components heading, paragraph from atmos/typography

File: src/components/about-tabs.js [hunk 8]
```diff
@@ -260,36 +263,61 @@ const AboutTabs = ({layout, columns}) => (
...
+            </Panel>
+
+            <Panel>
+              <PanelWrapper className="panel2" layout={layout}>
+                <h3>{content.content[6].header}</h3>
+                <p>{content.content[6].text}</p>
+
+                <NumericBoxedList content={content.content[6].items}/>
+
+                <p style={{ marginTop: 48 + "px" }}>All other steps associated with Incorporation apply to the creation of a nonprofit organization, such as paying fees, registering for business locally and <a href="https://www.incfile.com/manage-your-company/tax-id-ein/">applying for your EIN.</a></p>
```
Comment (Riotters) [inline src/components/about-tabs.js:310]: import, change to components

File: src/components/about-tabs.js [hunk 9]
```diff
@@ -260,36 +263,61 @@ const AboutTabs = ({layout, columns}) => (
...
+                <ArrowLink>Form your Nonprofit Corporation today!</ArrowLink>
+              </PanelWrapper>
+            </Panel>
+
+            <Panel>
+              <PanelWrapper className="panel2" layout={layout}>
+                <h3>{content.content[5].header}</h3>
+                <p>{content.content[5].text}</p>
+                <AcccordionCounting content={content.content[5]} tab/>
+                <p style={{ marginTop: 48 + "px" }}>Political parties are also eligible for tax exempt status, but do not fall under the standard nonprofit category.</p>
```
Comment (Riotters) [inline src/components/about-tabs.js:298]: import as text2

File: src/components/about-tabs.js [hunk 10]
```diff
@@ -260,36 +263,61 @@ const AboutTabs = ({layout, columns}) => (
...
-                <p>
-                  By having tax exempt status, your donations can be recorded by
-                  donors as not taxable by both the federal and state
-                  government. This is a great incentive to be able to offer your
-                  constituents.
-                </p>
+            <Panel>
+              <PanelWrapper className="panel2" layout={layout}>
+                <h3>{content.content[2].header}</h3>
+                <p>{content.content[2].text}</p>
```
Comment (Riotters) [inline src/components/about-tabs.js:277]: change to components

File: src/components/about-tabs.js [hunk 11]
```diff
@@ -260,36 +263,61 @@ const AboutTabs = ({layout, columns}) => (
...
+                <p>Another advantage is legal protection, as the founders and members are generally not held personally responsible for a nonprofit’s debts. The legal advantage is available without 501c3 status, as is the benefits that become available for employees, such as health and retirement plans. Furthermore, a nonprofit corporation that formalizes their existence improve the chances for longevity and success because the mission and vision must be clear and achievable.</p>
+                <p>There are few disadvantages of becoming a nonprofit. Nonprofits cost money and time to start. Once a nonprofit is started, you must keep excellent records and make sure to file all paperwork yearly. This can be resolved by hiring the proper professionals to handle such details. Since nonprofits are there to serve the public, they are also under greater public scrutiny.</p>
+                <h3>Don't get stuck, let us help you navigate the process.</h3>
```
Comment (Riotters) [inline src/components/about-tabs.js:288]: import, change to components

File: src/components/about-tabs.js [hunk 12]
```diff
@@ -260,36 +263,61 @@ const AboutTabs = ({layout, columns}) => (
...
+                <ArrowLink>Form your Nonprofit Corporation today!</ArrowLink>
+              </PanelWrapper>
+            </Panel>
+
+            <Panel>
+              <PanelWrapper className="panel2" layout={layout}>
+                <h3>{content.content[5].header}</h3>
+                <p>{content.content[5].text}</p>
```
Comment (Riotters) [inline src/components/about-tabs.js:296]: change to components

File: src/components/about-tabs.js [hunk 13]
```diff
@@ -260,36 +263,61 @@ const AboutTabs = ({layout, columns}) => (
...
+                <AcccordionCounting content={content.content[5]} tab/>
+                <p style={{ marginTop: 48 + "px" }}>Political parties are also eligible for tax exempt status, but do not fall under the standard nonprofit category.</p>
+                <ContentButton content={{text: `Download Our Start a Nonprofit Guide`, url: `#`}} theme="primary56" margin="0 auto 0 0" arrow />
+              </PanelWrapper>
+            </Panel>
+
+            <Panel>
+              <PanelWrapper className="panel2" layout={layout}>
+                <h3>{content.content[6].header}</h3>
+                <p>{content.content[6].text}</p>
```
Comment (Riotters) [inline src/components/about-tabs.js:306]: change to components

File: src/components/about-tabs.js [hunk 14]
```diff
@@ -260,36 +263,61 @@ const AboutTabs = ({layout, columns}) => (
...
+
+                <NumericBoxedList content={content.content[6].items}/>
+
+                <p style={{ marginTop: 48 + "px" }}>All other steps associated with Incorporation apply to the creation of a nonprofit organization, such as paying fees, registering for business locally and <a href="https://www.incfile.com/manage-your-company/tax-id-ein/">applying for your EIN.</a></p>
               </PanelWrapper>
             </Panel>
+
+            <Panel>
+              <PanelWrapper className="panel2" layout={layout}>
+                <h3>{content.content[7].header}</h3>
```
Comment (Riotters) [inline src/components/about-tabs.js:316]: change to components

**Review by Riotters — CHANGES_REQUESTED** [review]

**Review by Riotters — APPROVED** [review]

**Pull request merged**